templates/corporate/upgrade.html

```diff
@@ -24,6 +24,16 @@ <h1>🚀 {% trans %}Upgrade to {{ plan }}{% endtrans %}</h1>
                 </div>
                 {% endif %}
 
+                {% if is_demo_org %}
+                <div class="demo-org-warning">
+                    Demo organizations are not eligible for upgrades. Please
+                    <a href="/help/demo-organizations#converting-a-demo-organization-to-a-regular-organization">
+                        convert your demo organization
+                    </a>
+                    before upgrading.
```

Comment:
@alya can you propose a revision to this string?

Reply:
I think this billing/upgrade page commit is probably pretty close to mergeable, not having reviewed the CSS at all. I still think it's worth make it into its own independent PR (perhaps split into two commits, the first one adding Tippy configuration for the portico, and the second being the rest of this), since it's unrelated to the other changes in this PR, and it'll have some back-and-forth about wording and the like before it can be merged.

@laurynmm FYI.